server/vulnerabilities/macoffice/analyzer.go

```diff
@@ -81,10 +81,10 @@ func collectVulnerabilities(
 // getStoredVulnerabilities return all stored vulnerabilities for 'softwareID'
 func getStoredVulnerabilities(
 	ctx context.Context,
-	ds fleet.Datastore,
+	getSoftwareDBCall func(ctx context.Context, id uint, includeCVEScores bool) (*fleet.Software, error),
```

Comment:
How come you are changing the function signature but I can't see the change for the code that calls it?